### DISCUSSION TITLE
Update

### DISCUSSION THREAD

**Discussion by Abel-A16** [termux/termux-packages#11784] [Q&A] [answered]
Why  update is not available

**Comment by sylirre** [comment, marked as answer]
Could you clarify what exactly is not updated?

On side note:
* Termux application update is released once it is considered ready. We don't have update schedule there. Delay between updates can be from few weeks to a few months. For now the application is solely maintained by @agnostic-apollo where he has a free time for it.
* Packages are updated automatically when possible. If automatic update failed, we need to look into the issue and this may take a while. Some packages are not eligible for automatic updates and new version is published only if we aware about new upstream release and adjusted build recipes accordingly.
